transport_nantes/utm/templatetags/utm.py

```diff
@@ -0,0 +1,71 @@
+from django import template
+import datetime
+
+from django.db.models import Count, Max
+from utm.models import UTM
+
+register = template.Library()
+
+@register.inclusion_tag('utm/visit_overview.html', takes_context=True)
+def visit_overview(context):
+    """Provide a short summary of page visits.
+    """
+    request = context['request']
+    path = request.path
+    robots = UTM.objects.filter(base_url=path, ua_is_bot=True)
+    humans = UTM.objects.filter(base_url=path, ua_is_bot=False)
+    humans_unique = UTM.objects.filter(base_url=path, ua_is_bot=False) \
+                               .values('session_id').distinct()
+    context['robots_48h'] = robots.filter(
+        timestamp__gt=(datetime.datetime.now() - datetime.timedelta(hours=48))).\
+        count()
+    context['humans_48h'] = humans.filter(
+        timestamp__gt=(datetime.datetime.now() - datetime.timedelta(hours=48))).\
+        count()
+    context['humans_unique_48h'] = humans_unique.filter(
+        timestamp__gt=(datetime.datetime.now() - datetime.timedelta(hours=48))).\
+        count()
+
+    context['robots_7j'] = robots.filter(
+        timestamp__gt=(datetime.datetime.now() - datetime.timedelta(days=7))).\
+        count()
+    context['humans_7j'] = humans.filter(
+        timestamp__gt=(datetime.datetime.now() - datetime.timedelta(days=7))).\
+        count()
+    context['humans_unique_7j'] = humans_unique.filter(
+        timestamp__gt=(datetime.datetime.now() - datetime.timedelta(days=7))).\
+        count()
+
+    context['robots'] = robots.count()
+    context['humans'] = humans.count()
+    context['humans_unique'] = humans_unique.count()
+
+    sources = UTM.objects.filter(base_url=path, ua_is_bot=False) \
+                         .values('source').distinct()
+    print(sources)
+    context['sources'] = {}
+    for source in sources:
+        this_source = source['source']
+        print('source', this_source)
```

Comment:
Print left here, you can use the logger if this is relevant (the logger will also print in console while in dev)

Reply:
I don't see this at 9d14877

Reply:
Ah, I see, I could fix them in `HEAD^`.  Just a sec.